src/components/proposal/ProposalTableInstrumentScientist.tsx

```diff
@@ -398,19 +381,40 @@ const ProposalTableInstrumentScientist: React.FC<{
     }));
   }
 
+  if (
+    isTechnicalReviewEnabled &&
+    !columns.find((column) => column.field === 'technicalStatus')
```

Comment:
Why would there be no column `technicalStatus`?

Reply:
I think STFC are excluding that in their features